redwood/api/src/graphql/addressInformation.sdl.ts

```diff
@@ -0,0 +1,10 @@
+export const schema = gql`
+  type AddressInformation {
+    transactionCount: Int!
+    isFreshAddress: Boolean!
+  }
+
+  type Query {
+    fetchAddressTransactions(ethereumAddress: ID!): AddressInformation! @skipAuth
```

Comment:
You'll want to make sure you have eslint enabled in your editor to automatically fix all these warnings/errors on save. Alternatively I think you can run `yarn rw lint --fix` before committing.

Reply:
Is it ok if I add a pre-commit to handle this?

Reply:
Maybe you can open a PR with the proposed pre-commit and I can try it out?

I generally don't like heavy pre-commits because I make, amend and merge commits frequently locally to checkpoint my work. I then usually squash all that work into only a few commits that make it into the merged history.

Reply:
What about a pre-push then? :)

Reply:
Just yesterday I pushed a WIP branch in a state that wouldn't pass tests so that RoboTeddy could take a look at it. It would have been annoying to have to make the tests pass just to share my work. I guess maybe I'm missing some context here. Could you explain what problem adding checks as a pre-push would solve for you that the checks automatically run on a PR wouldn't solve?